pkgs/top-level/all-packages.nix

```diff
@@ -20803,6 +20806,14 @@ with pkgs;
 
   libcCross = assert stdenv.targetPlatform != stdenv.buildPlatform; libcCrossChooser stdenv.targetPlatform.libc;
 
+  threadsCross_pre_gcc_13 = if stdenv.targetPlatform.isMinGW && !(stdenv.targetPlatform.useLLVM or false)
+    then {
+      # other possible values: win32 or posix
+      model = "mcf";
+      # For win32 or posix set this to null
+      package = targetPackages.windows.mcfgthreads_pre_gcc_13 or windows.mcfgthreads_pre_gcc_13;
+    } else { };
```

Comment:
Please stop copy-pasting code like this.  Please.

https://github.com/NixOS/nixpkgs/pull/243394

Reply:
Feel free to revert. Thanks.

Reply:
I'm not asking for permission to revert this, I'm asking you to stop doing it.

When you duplicate code like this, eventually either:

1. Somebody will change one copy and forget to change the other.
2. Somebody will deliberately change only one, but leave no explanation of why they didn't change the other.

Either way, if the duplication hasn't been noticed before one of these happens, we don't know *which* of them happened.

Reply:
I consider a bit if I should stop contributing at all. Thanks.

Reply:
Removed myself from the org.

Reply:
@trofi please reconsider your decision, I don't think @amjoseph-nixpkgs has any ill intentions.

I know that it can be frustrating when higher standards for code are asked.

We all struggle between perfect and shipping, but let's leave the room for discussions.